common/common-rest/src/test/java/org/apache/servicecomb/common/rest/definition/TestRestOperationMeta.java

```diff
@@ -147,6 +147,18 @@ public void testCreateProduceProcessorsTextAndWildcard() {
             MediaType.APPLICATION_JSON + "," + MediaType.APPLICATION_XML + "," + MediaType.WILDCARD));
   }
 
+  @Test
+  public void testCreateProduceProcessorsWithSemicolon() {
```

Comment:
Can you add an integration tests for this? This is an user case, add an integration tests is better.